src/test/ui/span/multiline-span-simple.stderr

```diff
@@ -9,11 +9,7 @@ error[E0277]: the trait bound `u32: std::ops::Add<()>` is not satisfied
 27 | |             y),
    | |______________^ ...ending here: the trait `std::ops::Add<()>` is not implemented for `u32`
    |
-   = help: the following implementations were found:
-             <u32 as std::ops::Add>
-             <&'a u32 as std::ops::Add<u32>>
-             <u32 as std::ops::Add<&'a u32>>
-             <&'b u32 as std::ops::Add<&'a u32>>
+   = help: `u32 + ()` has no implementation
```

Comment:
I felt that for these cases in particular, in order to keep it newbie friendly, it was better to simplify the text in order to make the error less daunting.